mkopn/__init__.py

```diff
@@ -1,124 +1,78 @@
-import gzip
 import hashlib
 import math
-import io
 import pathlib
-import re
 import shutil
 import sys
 
 
 def main():
-    if len(sys.argv) != 4:
-        print(
-            f'Usage: {sys.argv[0]} COMPRESSED_IMAGE_FILE INSTALLER_FILE OPN_OUTPUT_FILE'
-        )
+    if len(sys.argv) != 3:
+        print(f'Usage: {sys.argv[0]} COMPRESSED_ROOT_IMAGE_FILE OPN_OUTPUT_FILE')
```

Comment:
My local YAPF is formatting this differently. What formatter are you using?

Reply:
Visual Studio Code Multicursor (aka manual formatting, looked better to me in the proposed version)

Reply:
Formatted with black now.